README.md

```diff
@@ -84,3 +84,31 @@ PLANNED ENHANCEMENTS:
 ----------------------
 - Add full sprintf_s() support
 - Add full sscanf_s() support
+
+
+# Compile and create Debian package (Ubuntu)
+On Ubuntu (probably also works on other Linux distributions), use the following commands to compile a library and create a Debian package for distribution.
+```
+mkdir build && cd build
+cmake ..
+make
+cpack
+```
+
+The generated package can be installed and removed using the following commands:
+```
+sudo dpkg -i libsafestring_1.2.0_amd64.deb
```

Comment:
Can you please, generalize the version, otherwise it will be confusing. Something like:
```
libsafestring_<version>_amd64.deb
```

Reply:
done